EIPS/eip-1815.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: 1815
+title: Interface for blind auctions
+author: Albert Acebrón (@corollari)
+discussions-to: https://github.com/ethereum/EIPs/issues/1745
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-03-04
+---
```

Comment:
Please add `requires: 165` if you are using ERC-165.

Reply:
Done! It slipped my mind.